Formula/keploy.rb

```diff
@@ -0,0 +1,36 @@
+require "language/node"
+require "fileutils"
+class Keploy < Formula
+  desc "Unit Testing Toolkit that creates test-cases and data mocks from API calls"
+  homepage "https://keploy.io"
+  url "https://github.com/keploy/keploy/archive/refs/tags/v0.5.5.tar.gz"
+  sha256 "37a4a27d30ed831c8fd4f60609df4c6b3ce1633fd468696e8098a35a16140377"
+  license "Apache-2.0"
+
+  depends_on "gatsby-cli" => :build
+  depends_on "go" => :build
+  depends_on "node@14" => :build
```

Comment:
This should use the latest node version

Reply:
@SMillerDev Is it compulsory to use the latest node version? 

What happens when the binary is incompatible with a newly released node version?

Reply:
Then it should be updated to use modern software.